gradle/libs.versions.toml

```
@@ -28,6 +32,18 @@ androidx-constraintlayout = { group = "androidx.constraintlayout", name = "const
 androidx-activity-compose = { module = "androidx.activity:activity-compose", version.ref = "androidx-activityCompose" }
 compose-ui-tooling = { module = "androidx.compose.ui:ui-tooling", version.ref = "compose" }
 compose-ui-tooling-preview = { module = "androidx.compose.ui:ui-tooling-preview", version.ref = "compose" }
+koin-bom = { module = "io.insert-koin:koin-bom", version.ref = "koin-bom" }
```

Al no usar `koin-bom` en otras dependencias, entonces es mejor evitar crear una variable innecesaria. 
Suggestion:
```suggestion
koin-bom = { module = "io.insert-koin:koin-bom", version= "3.5.6" }
```